security/monitoring.adoc

```diff
@@ -0,0 +1,204 @@
+[[security-monitoring]]
+= Monitoring Cluster Events and Logs
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[security-monitoring-intro]]
+== Introduction
+
+In addition to security measures mentioned in other sections of this
+guide, the ability to monitor and audit an {product-title} cluster is an
+important part of safeguarding the cluster and its users against
+inappropriate usage.
+
+There are two main sources of cluster-level information that
+are useful for this purpose: events and logs.
+
+[[security-monitoring-events]]
+== Cluster Events
+
+Cluster administrators are encouraged to familiarize themselves with the _Event_ resource
+type and review a xref:../dev_guide/events.adoc#events-reference[list of events] to
+determine which may be of interest. There are typically more potential
+event types than listed here, depending on master controller and plugin
+configuration.
+
+Events are associated with a namespace, either the namespace of the
+resource they are related to or, for cluster events, the *_default_*
+namespace. The *_default_* namespace is likely to hold the most relevant
+events for monitoring or auditing a cluster, both *_Node_* events and
+as resource events related to infrastructure components like the registry
```

Comment:
I'm confused by this (I wonder if a word is just missing):

" both *_Node_* events and as resource events related to infrastructure components like the registry and router."